### PR TITLE
Style cards main

### PR DESCRIPTION
**Pull request by ericweissman**
Added the border-box property for our entire app (for consistency/reduce dealing with margins, etc.) and made small tweaks to initial layout/structure for Header and the cards on the main view

main cards

### CODE REVIEW

**Review by foxwellm — APPROVED** [review]
Thanks for attaching the image

File: src/Main.scss [hunk 1]
```diff
@@ -1,3 +1,9 @@
 @import './variables';
 @import './containers/NoteCard/NoteCard.scss';
-@import './containers/App/App.scss';
\ No newline at end of file
+@import './containers/App/App.scss';
+@import './components/Header/Header.scss';
+
+* {
+  box-sizing: border-box;
+  margin: 0
```
Comment (geet084) [inline src/Main.scss:8]: nice catch, I don't think I would have remembered those!

File: src/containers/NoteCard/NoteCard.js [hunk 2]
```diff
@@ -7,25 +7,25 @@ export class NoteCard extends Component {
...
     const incomplete = this.props.noteItems.map((item) => {
       if (!item.isCompleted) {
-        return <p key={item.id}>{item.description}</p>
+        return <li key={item.id}>{item.description}</li>
       } else return []
     })
     const complete = this.props.noteItems.map((item) => {
       if (item.isCompleted) {
-        return <p key={item.id}>{item.description}</p>
+        return <li key={item.id}>{item.description} </li>
```
Comment (geet084) [inline src/containers/NoteCard/NoteCard.js:15]: I like that you changed them to LI items 👍

**Review by geet084 — APPROVED** [review]
Like Matt said, the image was a nice touch

**Pull request merged**